### PR TITLE
Update UI && Add API

### PR DESCRIPTION
**Pull request by MarsXue**
- Update SentryDetail UI
- Update AppFooter UI
- Update DashboardLayout UI
- Add sentry/update API
- Rename "wrong" to "invalid"

### CODE REVIEW

File: src/layouts/AppFooter.js [hunk 1]
```diff
@@ -11,7 +11,11 @@ class AppFooter extends React.Component {
       <Footer className="app-footer">
         Copyright <CopyrightOutlined /> WebSentry
         <br />
-        <a href="https://github.com/websentry/WebSentry">
+        <a
+          target="_blank"
+          rel="noopener noreferrer"
```
Comment (CaviarChen) [inline src/layouts/AppFooter.js:16]: Why?

**Review by CaviarChen — APPROVED** [review]
Overall is good, so the sentry/update api is added but it is not used yet?
Also is there any particular reason for using`rel="noopener noreferrer"`?

File: src/layouts/AppFooter.js (hunk 1 above)
Comment (MarsXue) [reply src/layouts/AppFooter.js:16]: image

**Comment by MarsXue** [comment]
> Overall is good, so the sentry/update api is added but it is not used yet?
> Also is there any particular reason for using`rel="noopener noreferrer"`?

UI for update is ongoing

**Pull request merged**